.pre-commit-config.yaml

```diff
@@ -20,6 +20,7 @@ repos:
   rev: v2.17.2
   hooks:
   - id: pylint
+    exclude: ^hack/boilerplate/boilerplate.py$
```

Comment:
I added this to make pylint ignore the linter errors reported on updating `boilerplate.py`